test/integration/admin_dashboard_direct_login_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.